### DISCUSSION TITLE
MQTT auto discovery for Home Assistant

### DISCUSSION THREAD

**Discussion by raybantracing** [sblantipodi/firefly_luciferin#93] [Q&A] [answered]
Hi,
have you ever considered adding MQTT auto discovery for Home Assistant?

Thanks

**Comment by sblantipodi** [comment, marked as answer]
This is something I have in mind since some time.

I have created a feature request just to track this request, it will be implemented in the next releases.

Thank you for using Luciferin and for sharing your ideas :)

**Comment by sblantipodi** [comment]
closing since the feature has been implemented
